pages/upload/upload.controller.js

```diff
@@ -42,7 +42,8 @@ function UploadController(api, session, $state, $translate, productType) {
 			self.loading = true;
 			api.upload(api.url + '/video', self.file, data, function() {
 				self.loading = false;
-				$state.go('gallery');
+				console.log('video updated, redirecting to userGallery');
```

Comment:
I'd say we don't need that console.log anymore =)

Reply:
Thanks.